lib/models/Food.js

```diff
@@ -32,6 +52,18 @@ module.exports = class Food{
     })
   }
 
+  static calorieSort(foods){
+    foods.sort((a,b)=>{
+      return (b.calories - a.calories)
+    })
+  }
+
+  static reverseCalorieSort(foods){
+    foods.sort((a,b)=>{
+      return (a.calories - b.calories)
+    })
+  }
+
```

Comment:
Every time I see this `sort` algorithm, I miss the built in sort functionality for Ruby. Haha!